mock/server/worker.go

```diff
@@ -70,11 +70,18 @@ const (
 	}`
 
 	// RegisterTokenResp represents a JSON return for an admin requesting a registration token.
-	//
-	//nolint:gosec // not actual credentials
+	// not actual credentials.
 	RegisterTokenResp = `{
 		"token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ3b3JrZXIiLCJpYXQiOjE1MTYyMzkwMjIsInRva2VuX3R5cGUiOiJXb3JrZXJSZWdpc3RlciJ9.gEzKaZB-sDd_gFCVF5uGo2mcf3iy9CrXDTLPZ6PTsTc"
 	}`
+
+	// QueueRegistrationResp represents a JSON return for an admin requesting a queue registration info.
+	//
+	// not actual credentials.
```

Comment:
note: surprised linter didn't bark here - if it does, add `//nolint:gosec // not actual credentials`

Reply:
odd that my linter was yelling at me for having `/nolint:gosec // not actual credentials` here. But i'll them back in